app/tests/navbar.component.ts

```diff
@@ -21,8 +21,20 @@ class NavBar {
     await testController.click('#login-dropdown-sign-in');
   }
 
+  async gotoCompanyListingPage(testController: TestController) {
+    await testController.click('#company-listing');
+  }
+
+  async gotoAddEventsPage(testController: TestController) {
+    await testController.click('#add-events-nav');
+  }
+
+  async gotoListEventsPage(testController: TestController) {
+    await testController.click('#list-events-nav');
+  }
+
   /** Check that the specified user is currently logged in. */
-  async isLoggedIn(testController, username) {
+  async isLoggedIn(testController: TestController, username) {
```

Comment:
I think it should have flagged you about `username` being an implicit any. Perhaps it should be a string?
```ts
username: string) {
```